.github/workflows/deploy-front-edge.yml

```diff
@@ -3,6 +3,15 @@ name: Deploy Front Edge
 on:
   workflow_dispatch:
     inputs:
+      regions:
+        description: "Regions to deploy to"
```

Comment:
Eventually we'll want to deploy to both regions right?

Reply:
Yup, added a "all" option in a next PR 😃